src/service/presence-api-service.js

```diff
@@ -1,14 +1,14 @@
 /**
  *
- * ## Presence API Adapter
+ * ## Presence API Service
  *
- * The Presence API Adapter provides methods to get and set the presence of a user in a project. It is based on query capabilities of the underlying RESTful [Presence API](../../../rest_apis/presence/).
+ * The Presence API Service provides methods to get and set the presence of an end user in a project, that is, to indicate whether the end user is online. This can be done explicitly: you can make a call to indicate that a particular end user is online or offline. This is also done automatically: in projects that use [channels](../epicenter-channel-manager/), the end user's presence is published automatically on a "presence" channel that is specific to each group.
```

Comment:
I'd switch the order around -- say this usually happens automatically for projects with the channels turned on, but can also be explicitly triggered by ..